org-code-javabuilder/media/src/main/java/org/code/media/FontHelper.java

```diff
@@ -0,0 +1,54 @@
+package org.code.media;
+
+import java.awt.*;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+import org.code.protocol.InternalErrorKey;
+import org.code.protocol.InternalServerRuntimeError;
+
+public class FontHelper {
+  private final Map<String, java.awt.Font> fontMap;
+
+  public FontHelper() {
+    this.fontMap = new HashMap<>();
+    this.populateFontMap();
+  }
+
+  public java.awt.Font getFont(Font font, FontStyle fontStyle) {
+    return this.fontMap.get(
+        fontToFilenamePrefix.get(font) + fontStyleToFilenameSuffix.get(fontStyle));
+  }
+
+  private void populateFontMap() {
+    try {
+      for (Font font : Font.values()) {
+        for (FontStyle fontStyle : FontStyle.values()) {
+          String filename =
+              fontToFilenamePrefix.get(font) + fontStyleToFilenameSuffix.get(fontStyle);
+          InputStream fileStream = this.getClass().getClassLoader().getResourceAsStream(filename);
+          this.fontMap.put(
+              filename, java.awt.Font.createFont(java.awt.Font.TRUETYPE_FONT, fileStream));
+        }
+      }
+    } catch (FontFormatException | IOException e) {
+      // throw an exception if we can't load a font, as we will hit a null pointer exception
+      // later if the user tries to use that font.
+      throw new InternalServerRuntimeError(InternalErrorKey.INTERNAL_RUNTIME_EXCEPTION);
```

Comment:
Also include the exception as the cause. i.e. 
`throw new InternalServerRuntimeError(InternalErrorKey.INTERNAL_RUNTIME_EXCEPTION, e);`

Reply:
done, thanks for catching!